app-admin/pwman3/pwman3-0.9.9.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_5 python3_6 )
+PYTHON_REQ_USE="sqlite?"
+inherit distutils-r1
+
+DESCRIPTION="A lightweight password manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="mongodb mysql postgres +sqlite"
+
+RDEPEND="
+	>=dev-python/cryptography-2.2.2[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.3.0[${PYTHON_USEDEP}]"
+
+DEPEND="
+	mongodb? ( dev-python/pymongo )
```

Comment:
Why are they only build-time deps? (why are they build-time deps in the first place?) Why did you remove PYTHON_USEDEP?!

Reply:
I am sorry, they where not there from the beginning.
Please accept my apology that it's taking so long. I was hasty, and I had really hard time understanding the comments you left. 

The tests were intentionally left out, because gentoo's policy is not to have any tests which need network services. I now modified the test suite to run without any networked service. Thus, the tests are only testing against SQLite.

I made a new release to PyPI, and this means that the current ebuild has a wrong version number.